web/src/components/organisms/TextAreaInput.jsx

```diff
@@ -0,0 +1,12 @@
+import { Label, Textarea } from 'flowbite-react';
+import React from 'react';
+
+const TextAreaInput = () => {
+    return (
+        <div className="col-start-1 col-end-auto">
+            <Textarea placeholder="Comment" rows={4} />
+        </div>
+    )
+}
```

Comment:
Think that this component will be used on other pages. Not all have a placeholder of comment
```suggestion
const TextAreaInput = ({ placeHolder }) => {
    return (
        <div className="col-start-1 col-end-auto">
            <Textarea placeholder={placeHolder} rows={4} />
        </div>
    )
}
```